app/controllers/employees_controller.rb

```diff
@@ -6,6 +6,12 @@ def new
   def create
     @employee = Employee.new(employee_params)
 
+    unless EmployeeFinder.new.employee_exists?(@employee.slack_username)
```

Comment:
Yes, definitely on both accounts.  I need to ban `unless` from my Ruby vocabulary... :-)